### DISCUSSION TITLE
User can choose the chat response attitude (example: "optimism"), and the magnitude of the attitude - between 0 (no effect) and 1(full effect)

### DISCUSSION THREAD

**Discussion by zaalgol** [oobabooga/text-generation-webui#6220] [Show and tell]
Hi, I added a UI and backend feature.
The user could enter a string that represents the chat response attitude (example: "optimism"), and the magnitude of the attitude - between 0 (no effect) and 1(full effect). I attached a screenshot.
![image](https://github.com/oobabooga/text-generation-webui/assets/21083266/01ca3264-78f9-4405-8e00-8154833092af)

The code was formatted and tested, and waiting for merge.
https://github.com/oobabooga/text-generation-webui/pull/6182

**Comment by TheLounger** [comment]
I like the idea of having an easy way to steer conversations in any direction you want, without having to edit replies. But as others have stated, this should be an extension.
It was also mentioned that a similar extension already exists: [character_bias](https://github.com/oobabooga/text-generation-webui/blob/main/extensions/character_bias/script.py). However, the last time I tried this (like two weeks ago) it was severely broken and made it impossible to have a normal conversation.
